tox.ini

```diff
@@ -37,8 +37,10 @@ commands_pre=
     devdeps: pip install -U --pre -i https://pypi.anaconda.org/liberfa/simple pyerfa
     devdeps: pip install -U --pre -i https://pypi.anaconda.org/astropy/simple astropy
 
+    # gwcs is incompatible with our oldest deps
+    oldestdeps: pip uninstall gwcs
```

Comment:
This means I need to make gwcs optional in the tests again. Which I can, I don't mind :shrug: